src/hardware/timers.cpp

```diff
@@ -68,7 +68,7 @@ void setupMotorTimers() {
     #endif
 
     // Attach the interupt to the step pin (subpriority is set in platformio config file)
-    attachInterrupt(digitalPinToInterrupt(STEP_PIN), stepMotor, RISING);
+    attachInterrupt(digitalPinToInterrupt(STEP_PIN), stepMotor, FALLING); // input is pull-upped to VDD
```

Comment:
Are you sure this is right? I think that I checked, the pin is supposed to trigger on the rising phase

Reply:
Only waiting on this, then I'll merge

Reply:
![image](https://user-images.githubusercontent.com/70886343/124286724-e96ffb80-db57-11eb-9d41-ffaea1f18ee0.png)
When STP_IN is not connected, then the upper transistor of U3(which is connected to pins 6-8) is closed and R15 pull-ups PA0 to VDD(HIGH level).
When STP_IN is connected and sett to HIGH level (incoming STEP - RISING), then the upper transistor of U3 will open and PA0 will go LOW level (FALLING). STM32 must catch the FALLING on PA0.
U3 works as level invertor.
Also tested on board.

Reply:
Excellent, thanks for explaining it!

Reply:
I added a little note and a link to this comment so that it won't be forgotten in the future